tests/unit/test_snapshot.py

```diff
@@ -1225,12 +1230,18 @@ def test_partition_read_other_error(self):
         with self.assertRaises(RuntimeError):
             list(derived.partition_read(TABLE_NAME, COLUMNS, keyset))
 
+        if not HAS_OPENTELEMETRY_INSTALLED:
+            return
```

Comment:
This is not needed right since `assertSpanAttributes ` method already checks for this condition?

Reply:
Yeah, before I was manually comparing attributes and span names but with the update that I made I no longer need it, done, thanks!